backend-tests/tests/test_azure.py

```diff
@@ -30,40 +37,62 @@
 )
 from testutils.api.client import ApiClient
 from testutils.common import (
-    Authset,
+    Device,
+    Tenant,
+    User,
     create_org,
     create_user,
     clean_mongo,
     make_accepted_device,
-    mongo_cleanup,
     mongo,
 )
-from testutils.infra.container_manager.docker_manager import DockerNamespace
-from testutils.infra.container_manager.kubernetes_manager import (
-    KubernetesNamespace,
-    isK8S,
+
+
+HTTPServer.DEFAULT_LISTEN_PORT = 8888
+HTTPServer.DEFAULT_LISTEN_HOST = (
+    "mender-backend-tests-runner"  # name of the compose service
 )
 
 
-def create_tenant_test_setup(user_name, tenant_name, password, plan="enterprise"):
-    """
-    Creates a tenant, and a user belonging to the tenant
-    """
+@pytest.fixture(scope="session")
+def ca():
+    return trustme.CA()
```

Comment:
Currently it's not trusted, Fabio convinced me to resign from mocking Azure, but I decided to leave this piece of code; maybe it will come in handy someday.